setup.sh

```diff
@@ -39,100 +39,61 @@ done
-FPM_FLAG="-fcoarray=single -O3 -fallow-argument-mismatch -ffree-line-length-none -L$NETCDF_LIB_PATH -L$HDF5_LIB_PATH"
 FPM_FC=${FC:-"gfortran-13"}
 FPM_CC=${CC:-"gcc-13"}
 
 mkdir -p build
 
-CI=${CI:-"false"} # GitHub Actions workflows set CI=true
-
-if [ $CI = true ]; then
-  PKG_CONFIG_PATH=`realpath ./build/pkgconfig`
-  echo "---------------"
-  echo "PKG_CONFIG_PATH=$PKG_CONFIG_PATH"
-  echo "---------------"
-else
-  PKG_CONFIG_PATH=`realpath "$PREFIX"/lib/pkgconfig`
-fi
-
-if [ ! -d $PKG_CONFIG_PATH ]; then
-  mkdir -p $PKG_CONFIG_PATH 
-fi
-
-INFERENCE_ENGINE_PC="$PKG_CONFIG_PATH/inference-engine.pc"
-echo "INFERENCE_ENGINE_FPM_CC=\"$FPM_CC\""                  >  $INFERENCE_ENGINE_PC
-echo "INFERENCE_ENGINE_FPM_FC=\"$FPM_FC\""                  >> $INFERENCE_ENGINE_PC
-echo "INFERENCE_ENGINE_FPM_LD_FLAG=\"$FPM_LD_FLAG\""        >> $INFERENCE_ENGINE_PC
-echo "INFERENCE_ENGINE_FPM_FLAG=\"$FPM_FLAG\""              >> $INFERENCE_ENGINE_PC
-echo "Name: inference-engine"                               >> $INFERENCE_ENGINE_PC
-echo "Description: Inference Engine"                        >> $INFERENCE_ENGINE_PC
-echo "URL: https://github.com/berkeleylab/inference-engine" >> $INFERENCE_ENGINE_PC
-echo "Version: 0.1.2"                                       >> $INFERENCE_ENGINE_PC
-if [ $CI = true ]; then
-  echo "---------------"
-  echo "cat $INFERENCE_ENGINE_PC"
-  cat $INFERENCE_ENGINE_PC
-  echo "---------------"
-fi
-
-export PKG_CONFIG_PATH
-cp scripts/run-fpm.sh-header build/run-fpm.sh
-RUN_FPM_SH="`realpath ./build/run-fpm.sh`"
-echo "`which fpm` \$fpm_arguments \\" >>  $RUN_FPM_SH
-echo "--profile release \\" >> $RUN_FPM_SH
-echo "--c-compiler \"`pkg-config inference-engine --variable=INFERENCE_ENGINE_FPM_CC`\" \\" >> $RUN_FPM_SH
-echo "--compiler \"`pkg-config inference-engine --variable=INFERENCE_ENGINE_FPM_FC`\" \\" >> $RUN_FPM_SH
-echo "--flag \"-cpp `pkg-config inference-engine --variable=INFERENCE_ENGINE_FPM_FLAG`\" \\"  >> $RUN_FPM_SH
-echo "--link-flag \"`pkg-config inference-engine --variable=INFERENCE_ENGINE_FPM_LD_FLAG`\" \\" >> $RUN_FPM_SH
-echo "\$program_arguments" >> $RUN_FPM_SH
-chmod u+x $RUN_FPM_SH
-if [ $CI = true ]; then
-  echo "---------------"
-  echo "cat $RUN_FPM_SH"
-  cat $RUN_FPM_SH
-  echo "---------------"
-fi
-
-if command -v fpm > /dev/null 2>&1; then
-  brew tap fortran-lang/fortran
-  brew install fpm
-fi
-
-echo "$RUN_FPM_SH test"
-$RUN_FPM_SH test
+fpm test
 
 echo ""
-echo "____________________ Inference-Engine has been set up! _______________________" 
+echo "____________________ Inference-Engine has been set up! _______________________"
 echo ""
 echo "To run one of the programs in the example subdirectory, enter a command of the"
 echo "following form at a shell command prompt after replacing <example-base-name>"
 echo "with the base name of a file in the example/ subdirectory:"
 echo ""
-echo "./build/run-fpm.sh run --example <example-base-name>"
+echo "fpm run --example <example-base-name>"
```

Comment:
I'm thinking we should start adding `--profile release` to get optimized builds.  
```suggestion
echo "fpm run --example <example-base-name> --profile release"
```
Let's add similar text in the README.md.  That can be in a new PR.